pkgs/tools/X11/kdocker/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, lib, qtbase, wrapQtAppsHook, fetchFromGitHub, qmake, qtx11extras, libX11, libXmu, libXpm }:
```

Comment:
```suggestion
{ stdenv
, lib
, qtbase
, wrapQtAppsHook
, fetchFromGitHub
, qmake
, qtx11extras
, libX11
, libXmu
, libXpm
}:

```

Reply:
Done! Also fixed a mistake in the patching phase.